src/main/kotlin/no/nav/veilarbaktivitet/motesms/MoteSMSService.kt

```diff
@@ -0,0 +1,117 @@
+                if (brukernotifikasjonService.kanVarsles(it.aktorId)) {
+                    val varsel = AktivitetVarsel(
+                        it.aktivitetId,
+                        it.aktitetVersion,
+                        it.aktorId,
+                        it.getDitNavTekst(),
+                        VarselType.MOTE_SMS,
+                        it.getEpostTitel(),
+                        it.getEpostBody(),
+                        it.getSmsTekst()
+                    )
+                    val varselId = brukernotifikasjonService.opprettVarselPaaAktivitet(varsel)
+                    moteSmsDAO.insertGjeldendeSms(it, varselId)
+                } else {
+                    // Usikker på hvorfor man inserter i gjeldende sms når bruker ikke kan
+                    // varsles men det var gjort slik tidligere
+                    moteSmsDAO.insertGjeldendeSms(it, null)
+                    log.info(
+                        "Minside varsel ikke sendt (møte sms), bruker kan ikke varsles {}",
+                        it.aktorId
+                    )
+                }
+            })
+    }
+
+    @Scheduled(
+        initialDelayString = "\${app.env.scheduled.default.initialDelay}",
+        fixedDelayString = "\${app.env.scheduled.default.fixedDelay}"
+    )
+    @SchedulerLock(name = "stopp_mote_sms_scheduledTask", lockAtMostFor = "PT20M")
+    @Timed(value = "stopmoteservicemelding", histogram = true)
+    open fun stopMoteSms() {
+        MDC.put("running.job", "moteSmsServiceStopper")
+        stopMoteSms(5000)
+        MDC.clear()
+    }
+
+    open fun stopMoteSms(maxAntall: Int) {
+        val batchResult = batchTrackingDAO.withOffset(BatchJob.StoppMoteSms) {
+                sisteProsesserteVersjonFørBatch ->
+            moteSmsDAO.hentMoterMedOppdatertTidEllerKanal(maxAntall, sisteProsesserteVersjonFørBatch)
+                .map { aktivitetVersjon: AktivitetVersjon ->
+
+                    try {
+                        brukernotifikasjonService.setDone(aktivitetVersjon.aktivitetId, VarselType.MOTE_SMS)
+                        moteSmsDAO.slettGjeldende(aktivitetVersjon.aktivitetId) //TODO endre til send beskjed sms om flyttet møte + skal sende på nytt hvis møtet er mere enn 48 timer fremm i tid
+                        return@map BatchResult.Success(aktivitetVersjon.versjon)
+                    } catch (e: Exception) {
+                        log.warn("Feil under prosessering av StoppMøteSms batch", e)
+                        return@map BatchResult.Failure(aktivitetVersjon.versjon)
+                    }
+                }
+        }
+        if (batchResult.size > 0) log.info("Behandlet stoppMøteSms for {} møter som har oppdatert tid eller kanal", batchResult.size)
+        moteSmsDAO.hentMoteSmsSomFantStedForMerEnd(Duration.ofDays(7)) //TODO Trenger vi denne? Holder det at bruker kan fjerne den og den forsvinner når aktiviteter er fulført/avbrut eller blir historisk
+            .forEach(Consumer { it: Long ->
```

Comment:
```suggestion
            .forEach {
```

Trenger ikke `Consumer` i Kotlin og hvis parameteret heter it så trenger man ikke ta det inn. Ofte trenger man ikke deklarere typen heller